### ISSUE TITLE
Error when 301 response missing Location header

### ISSUE THREAD

**Issue by Techbrunch** [ffuf/ffuf#198]
Hello,

First thanks for this awesome tool :)

I ran into an issue where my target would return 301 response without a location header which triggered an error in ffuf.

```
:: Progress: [1/1] :: Job [1/1] :: 0 req/sec :: Duration: [0:00:00] :: Errors: 1 ::
```

```
2020/03/27 16:37:49 Head https://target.com/word: 301 response missing Location header
```

I would expect this to not be a problem if I don't set the Follow redirects flag to true and to match according to Matchers.

Also I don't know if this is bug but in this situation ffuf sends 2 requests instead of one.

**Comment by joohoi** [comment]
Thanks for opening the issue! This should get fixed for the next release.

> Also I don't know if this is bug but in this situation ffuf sends 2 requests instead of one.

ffuf retries the payloads in case of errors (which this should not have been obviously).

**Comment by Techbrunch** [comment]
@joohoi Looks like this issue was fixed with #265 and that #264 is a duplicate of #198

**Comment by joohoi** [comment]
Closing the issue in order to keep the issues backlog readable, and I do consider the issue either resolved or there's no communication. If you have something to add to the issue, please comment and we'll consider reopening it.
